addons/analytic/tests/common.py

```diff
@@ -41,3 +41,9 @@ def setup_independent_company(cls, **kwargs):
     def get_default_groups(cls):
         groups = super().get_default_groups()
         return groups + cls.env.ref('analytic.group_analytic_accounting')
+
+    @classmethod
+    def _enable_analytic_accounting(cls):
+        cls.env.ref('base.group_user').sudo().write({'implied_ids': [
```

Comment:
Nitpicking and suggestion: Since there is only one element maybe we can directly do
```
cls.env.ref('base.group_user').sudo().implied_ids = ...
```
Also i prefer to use the Command than just a number, i find that more clear

Reply:
Good suggestion, thanks! I can see that VFE left a few comments as well, so possibly I'll change the way that groups are assigned. But if not, I will switch to Command / use `implied_ids` directly :+1:

Reply:
I've updated the code and now groups are assigned directly to the user ;)